### PR TITLE
Local storage saving data from contact form

### PR DESCRIPTION
**Pull request by gjuliao**
In one object, it saves the name, email, and message of the person submitting the form.
If the user adds new information, it changes and saves it.
It saves the information when the page reloads.

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @gjuliao, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199


You have submitted a pair programming project with commit from only one partner. Kindly ensure that every member of the team participates in the project evidenced by their commits. If you're having issues making contacts with any of the team members, kindly reach out to the Student Success team.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -130,6 +130,24 @@ document.getElementById('contact-form').addEventListener('submit', (e) => {
...
+    "name" : nameLocal,
+    "email" : emailLocal,
+    "bodyLocal" : bodyLocal,
+  }
+
+  window.localStorage.setItem("pData", JSON.stringify(pData));
+}
+
+
+
```
Comment (Qoosim) [inline script.js:150]: - [ ] There're some project requirements you need to implement here before you project can be due for approval.
- After filling the form, and not submitting it, the entered values should be saved in the browser. This behaviour is failing in your project. Kindly look into it and fix. Take a look :point_down: 

### Your Project Behaviour.


https://user-images.githubusercontent.com/26597432/202740777-41b9cf27-5145-4b4b-8c39-5029f9cc33a0.mp4

- [ ] When visiting the page the form inputs should be contained, be auto-populated with, the values entered in the previous visit to the page. After filling the form and clicked submit button, I was taken to the formspree. I came back to your page and reloaded the page but the previously inputed data got disappeared. This should not be so, the values on your form should remain there on page load. Take a look. :point_down: 

### Your Form Behaviour


https://user-images.githubusercontent.com/26597432/202741732-d348efe1-43f6-4e86-a6e4-c9ecc1243b8d.mp4

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Kindly fill in all the appropriate sections of your README file.

**Review by OyePriscilla — APPROVED** [review]
Hi @gjuliao and @roodjinocherilus , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Descriptive pull request message and title :heavy_check_mark: 
Linter error checks validated :heavy_check_mark: 
Commit messages are meaningful. :heavy_check_mark: 
GitHub flow observed ✔️ 

 ## Required Changes ♻️
- [ ] As the previous reviewer pointed out, please kindly ensure that the data stored in the local storage persist after you refresh your browser. 
## I input my contact details and got this 👇  after I refreshed the browser.

sto

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @gjuliao, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Data persisted in local storage ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**